openmetadata-service/src/main/resources/json/data/ResourceDescriptors.json

```diff
@@ -57,6 +57,16 @@
       "EditLineage"
     ]
   },
+  {
+    "name" : "query",
+    "operations" : [
+      "Create",
+      "Delete",
+      "ViewAll",
+      "ViewQueries",
+      "EditAll"
```

Comment:
Will queries have description, etc? In which case Edit operations will need to be granular.

Reply:
yes it will have Descriptions